connectivity/conn_http.c

```diff
@@ -231,6 +231,8 @@ retcode_t set_get_request(char const *const path, char const *const host, const
 }
 
 int parser_body_callback(http_parser *parser, const char *at, size_t length) {
-  http_res_body = strdup(at);
+#ifdef DEBUG
+  printf("HTTP Response: %s\n", at);
```

Comment:
Is it equivalent for dumping debug message?

Reply:
Yes. But the `http_res_body` will not use anymore.